playbooks/openshift-etcd/private/config.yml

```diff
@@ -29,6 +29,10 @@
     - openshift_is_atomic | bool
     - not inventory_hostname in groups['oo_masters']
 
+  - import_role:
```

Comment:
It's not clear to me why this is needed.

The scaleup play already calls this for 'new member'.

What scenario triggers this problem?

Reply:
Minor version upgrade would throw "verify Openshift is installed" without this